scrapy/exporters.py

```diff
@@ -99,7 +100,7 @@ def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
         kwargs.setdefault('ensure_ascii', not self.encoding)
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        self.encoder = ScrapyJSONEncoder(indent=self.indent_width, **kwargs)
```

Comment:
This looks backwards incompatible for people who passed `indent` argument for JsonItemExporter - I think it'll now raise a TypeError exception `got multiple values for keyword argument 'indent'`. For backwards compatibility it'd be great to handle this use case; `indent` in kwargs should take precedence over `indent_width`.